Tests/AuthTests/GoTrueClientTests.swift

```diff
@@ -31,11 +39,11 @@ final class AuthClientTests: XCTestCase {
             $0.append(event)
           }
 
-          expectation.fulfill()
+          semaphore.signal()
```

Comment:
Using a semaphore since it was the best cross-platform tool to wait for some async changes